src/librepgp/stream-parse.cpp

```diff
@@ -797,40 +810,56 @@ signed_src_update(pgp_source_t *src, const void *buf, size_t len)
     if (param->txt_hashes.empty()) {
         return;
     }
-    /* check whether we had CR at the end of last chunk and LF at the beginning */
+
     uint8_t *ch = (uint8_t *) buf;
-    if (param->lastcr && (*ch == CH_LF)) {
-        ch++;
-    }
     uint8_t *linebeg = ch;
     uint8_t *end = (uint8_t *) buf + len;
-    /* we support CR, LF and CRLF line endings */
+    bool     max_linelen_warn = false;
+    /* we support LF and CRLF line endings */
     while (ch < end) {
-        /* continue if not reached CR or LF */
-        if ((*ch != CH_CR) && (*ch != CH_LF)) {
+        /* continue if not reached LF */
+        if (*ch != CH_LF) {
+            if (*ch != CH_CR && param->stripped_crs > 0) {
+                while (param->stripped_crs--)
+                    pgp_hash_list_update(param->txt_hashes, ST_CR, 1);
```

Comment:
Please add `{}` around the `pgp_hash_list_update()`

Reply:
done